ipn/ipnlocal/ssh.go

```diff
@@ -32,6 +38,70 @@ import (
 // running as root.
 var keyTypes = []string{"rsa", "ecdsa", "ed25519"}
 
+func (b *LocalBackend) getSSHUsernames() (*tailcfg.C2NSSHUsernamesResponse, error) {
+	res := new(tailcfg.C2NSSHUsernamesResponse)
+
+	b.mu.Lock()
+	defer b.mu.Unlock()
+
+	if b.sshServer == nil {
+		return res, nil
+	}
+
+	add := func(u string) {
+		switch u {
+		case "nobody", "daemon", "sync":
```

Comment:
Other things we may want to denylist that IIRC usually have a shell set: `mysql`, `postgres`.

Reply:
They have shells set? Curious why!

Reply:
No idea! But on my home (NixOS) server:
```
postgres:x:71:71:PostgreSQL server user:/var/lib/postgresql/13:/run/current-system/sw/bin/bash
```

Reply:
Instead of hard coding it, maybe we can send a filter as a query param and only return unmatching usernames. Helps us iterate without bumping clients

Reply:
I mean, we can also ignore things server-side.

Reply:
We don't send the full user list, so it is plausible that none of the names are useful.

Reply:
I don't want to filter mysql or postgres by default: it's possible that you're running a database server and your only non-root user account is mysql or postgres.

If there's a better heuristic to exclude it, I'm all ears.